.eslintrc.cjs

```diff
@@ -12,17 +12,33 @@ module.exports = {
 	overrides: [
 		{
 			extends: ["plugin:markdown/recommended"],
-			files: ["**/*.{md}"],
+			files: ["**/*.md", "**/*.md/*.js"],
 			processor: "markdown/markdown",
 		},
+		{
+			// https://github.com/eslint/eslint-plugin-markdown/issues/114#issuecomment-818463890
+			files: ["**/*.md/*.ts"],
+			parser: "@typescript-eslint/parser",
+			// Only rules that don't require type information are allowed
+			extends: [
+				"plugin:@typescript-eslint/recommended",
+				"plugin:typescript-sort-keys/recommended",
+			],
+			rules: {
+				"@typescript-eslint/no-unused-vars": "off",
+				"@typescript-eslint/padding-line-between-statements": "off",
+			},
+		},
 		{
 			extends: [
 				"plugin:@typescript-eslint/recommended",
 				"plugin:@typescript-eslint/recommended-requiring-type-checking",
 				"plugin:typescript-sort-keys/recommended",
```

Comment:
[Cleanup] It's not ideal duplicating these `extends` values in two overrides, especially since one override is a superset of the other. Let's delete everything from this one (that deals with type-system-capable files) that is already enabled in the first one (that deals with all `.ts` files).

```suggestion
				"plugin:@typescript-eslint/recommended-requiring-type-checking",
```

Reply:
Removed duplicates, let me know if duplicated `files: ["**/*.ts"]` is fine.

Reply:
Yeah I don't see a way around it. 👍